dla_cnn/desi/model_v2.1.py

```diff
@@ -0,0 +1,243 @@
+""" TensorFlow Models for DLA finder"""
```

Comment:
While it is great to see this code, please take it out of
this PR, put it in its own branch and then submit a PR 
for it when it is ready.